README.markdown

```diff
@@ -384,152 +319,111 @@ This example will provision a PE Agent and will point it to your PE master.
    },
 ```
 
-### Implementation of classification
+##Development
 
-In addition to creating instances with `gce_instance`, you may pass additional parameters to configure and classify the instance. The work is done during instance creation by a bootstrap script. The module includes a script to configure open source Puppet and another script for Puppet Enterprise.
+To setup a development environment, follow the [Setup](#setup) instructions above, up until
 
-In the `gce_instance` resource, you may provide the following parameter to choose a startup script. You can use any executable script that's located in the gce_compute modules files directory and can be interpreted by the OS GCE provisisions.
-
-```puppet
-   startupscript => 'script_to_use.sh'
-   startupscript => 'puppet-community.sh'
-   startupscript => 'puppet-enterprise.sh'
+```bash
+$ puppet module install puppetlabs-gce_compute
 ```
 
-You can pass additional parameters to `gce_instance` resources to influence the behavior of these startup scripts. Both included scripts are capable of installing Puppet, classifying into a Dashboard/Console ENC, and installing modules. See the `gce_instance` reference (above) for more details.
+Instead, clone this repository, `cd` into the repository, then do
 
-Common Parameters.
+```bash
+$ rake install
```

Comment:
As noted elsewhere, this shouldn't be required. puppetlabs_spec_helper should do all the work but make a global install not required.